### DISCUSSION TITLE
video-downloadhelper & eroprofile.com

### DISCUSSION THREAD

**Discussion by Carciofo70** [aclap-dev/video-downloadhelper#87] [Help and Q&A]
### Your question

Hello everyone. After updating video-downloadhelper to the latest version (automatically) I can no longer download videos from eroprofile.com... does anyone have any suggestions? Thank you.

**Comment by GrampaWildWilly** [comment]
I suggest you post a proper problem report.  That includes filling in the version & details section, as well as providing at least one URL of a web page not requiring a log in on which a video is hosted that exhibits the problem you are having.  A series of screenshots might shorten your typing & be clearer anyway.  In this forum, unlike our old forum on Google, it is possible to edit your own posts.  Just click the 3 dots above & to the right of your post.  You'll get the hang of it quickly enough.  This should help you: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by paulrouget** [comment]
Hmm, interesting, the video don't show up in the popup, but they show in the context menu.

I'll investigate.

**Comment by paulrouget** [comment]
I found the issue. I'm planning to push a new version in the coming days (8.2.0.24) with a fix.

**Reply by Carciofo70** [reply]
Thank you very much! I'm anxiously waiting...

**Reply by git7878** [reply]
I'm having the exact same problem.  Will the new version with the fix come automatically?
Thank you for all your hard work.

**Reply by paulrouget** [reply]
It will update automatically.

**Reply by Carciofo70** [reply]
Thank you for your work!

**Reply by Carciofo70** [reply]
@paulrouget 👍 
I don't know how to thank you! It started working properly again! Thanks again and respect!

**Reply by paulrouget** [reply]
Happy it makes you happy :)

**Reply by Carciofo70** [reply]
https://github.com/paulrouget
Good morning Paul. Today the same problem as in February 2024 affects this site: https://thisvid.com/. I attach screenshots of the problem. The version is the latest: 9.1.0.43 (stable). 
![Immagine 2024-08-26 080314](https://github.com/user-attachments/assets/59365727-c6ea-4074-a925-c19e645f6e53)
Thank you  for your interest.

**Reply by GrampaWildWilly** [reply]
Just to be complete, let me say again that I do not work for VDH.  I'm just another user.

Since this thread was launched, there have been many updates to my system.  I am now running Windows 11 Pro 64-bit version 23H2 build 22631.4037, Firefox 129.0.2 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

I am running the VDH beta.

Have I mentioned that I am running the VDH beta?

I don't want anyone to miss the fact that I am running the VDH beta.

I have often advised people to get the VDH beta but they don't seem to have noticed that.

So I am going to make it extremely obvious.  I am running the VDH beta.  You should be running it as well.

You should always post your sample URL in plain text within your post.  Like this:

https://thisvid.com/videos/hot-jeans-wetting9/

That way, you are making it easy for somebody else to help you.  At least the URL was clear in the image you posted.

When I first visited your web page, VDH recognized nothing.  So I clicked the play button in the video player on the web page & paused it after 3 seconds.  VDH recognized this:

![#01](https://github.com/user-attachments/assets/19a227d2-2c5e-43f5-863d-b7f84c1e46f2)

The video is only about 9 minutes long so I decided to download both those items.  It is kind of annoying that there's no duration quoted in the information VDH displays.  If there is a problem to report, it is that VDH isn't giving proper information about the video on this page.  In any case, I downloaded both items:

![#02](https://github.com/user-attachments/assets/4bafbb9b-4769-4ad2-b15a-dc3359471224)

Those are simply atrociously slow download speeds.  It is interesting that VDH shows a percentage complete on the first item but only an elapsed time of download for the second item.  It seems VDH recognized the second item as a livestream.  It's not a livestream but VDH treated it like one.  I suppose that is a result of the way the web site presents the information about the video to VDH.

Here are the notifications I got from when the 2 downloads completed, which happened to be at almost the same time:

![#03](https://github.com/user-attachments/assets/56dd9743-870c-4eeb-9344-37c84c22c242)

Here's what VDH told me after the downloads completed.  Notice that I changed the names so I could tell the 2 files apart:

![#04](https://github.com/user-attachments/assets/3f7bb645-f733-4166-9865-a9bb6a41ab40)

This is what the first download got for me:

![#05](https://github.com/user-attachments/assets/2c684896-d896-4415-9a70-053baae5d3cd)

And here is the same information for the second download:

![#06](https://github.com/user-attachments/assets/d1850909-ad19-48ff-995b-2c229ee63cfd)

Looks like the same video.  When I played them in VLC, they turned out to indeed be the same video.

Conclusion: You should be running the VDH beta.  It works for me.  I'm sure it will work for you.

One rather minor footnote.  I did not sit & watch this, just skimmed it, sampling it at intervals.  There seems to be some fault in the source material shortly after time index 7:00.  This is in the source.  VDH did not introduce this video fault.  It is in the source material on the web site.  You can see the video fault if you play the video in the player on the web page.  VDH simply downloaded what was there, errors & all.

**Reply by Carciofo70** [reply]
Thanks for the quick and kind response. I thought about contacting Paul directly but evidently I don't know how to use this platform correctly to communicate. The fact is that I don't use Firefox but Vivaldi and in any case even on Edge and even on different PCs I always encounter the same thing that I illustrated for that site. This with all browsers available. It is likely at this point that the problem is in the VDH version which, despite being updated, has something wrong... The fact remains that the problem is only aimed at thisvid.com and no one else.

**Reply by GrampaWildWilly** [reply]
The only Vivaldi I know is Antonio Vivaldi of "The Four Seasons" fame.  You surely know that VDH is supported only on Firefox, Chrome, & Edge, of which I know only Firefox.  I'm not clear on what exactly you are illustrating in your image you posted earlier.  It doesn't quite match what I got.  In any case, I am using the VDH beta.  You need to go get the beta & see if that makes any difference.  Don't rely on automatic updates.  In fact, I have recommended you disable automatic updates.  There is an issue that if the browser decides to update VDH (or any other browser extension, for that matter) while VDH is in use, the browser goes ahead & updates VDH.  Note that.  The browser decides when to install automatic updates, not the extension, not the author of the extension, the browser.  This would cause any downloads you have in progress to fail.  They would have to be done over from the beginning.  So you should turn off automatic updates.  Then you should go into the Wiki, find the beta for VDH, & install it.  You haven't posted your information from the `Copy information to clipboard` button in the VDH settings.  I've been assuming you are using the VDH for Firefox.  If that's not the case, then you're kind of out of luck because there is a VDH beta only for Firefox.  My experience shows that your video is easily downloaded by VDH for Firefox.  If you're using something else, then you're ignoring the evidence that Firefox works fine.

When it comes to contacting Paul, posting here is how you do it.  Paul participates here.  I don't know that he participates here as much as he should, but he does participate here.  Posting here is supposed to be all that is necessary to catch his attention.  Supposed to be.

**Comment by git7878** [comment]
Hi Paul,
I read that the other person in this thread has gotten the fix and has full
use of his downloading function now, but I'm sorry to say mine still
doesn't work.  Do the fixes come out at different times or ?.
Thanks very much,
Tom

**Reply by paulrouget** [reply]
@git7878 I need details, otherwise I can't help.

Please share the URL of the video you're trying to download, and share your version like this: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by git7878** [comment]
OK Paul,
It's the same site as Carciofo70 had problems with and here is the link:

https://eroprofile.com/m/videos/view/Hot-sexy-sensual-dancing-Latin-Style

Thank you very much.

**Reply by GrampaWildWilly** [reply]
I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3155, Firefox 123.0 64-bit, licensed VDH 8.2.1.3a1 beta, CoApp 2.0.19.  And I don't work for VDH.  I'm just another user.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/654f050c-fa72-4dc8-9d16-cc8b47ebaecf)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/7695a053-13a6-434d-9aec-9ccba08304a1)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8c3bf2ef-7ec0-4ea5-9a67-cc87b52890cc)
![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/5d9d2e73-e2c0-41a2-914e-4c0f54d1de53)
![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/b785b1a8-ad51-4b25-af09-55ccbed1be1e)

The first & third variants were ads.  The second variant was the dance video.  I don't know where m4v comes from but VLC played it just fine, as it did both ads.

Now if there's a complaint here, it's that the variants don't give any clue what they are.  Durations would have helped a great deal.  If the middle one had showed the duration 4:14, we would know that was the dance video.  I suppose the size 18.8M on the middle variant could be a hint.  But it's only a hint.  18.8M is not very big.  The resolution I got with that one was 480x360, quite low, which kind of explains the small file size.  But since resolution also isn't shown on the VDH menu, you are left to guess.

Having said that, I understand that these bits of information are not necessarily always available.  I haven't bothered to investigate what the Network Monitor would show me.  Maybe there's more information in there that VDH might take advantage of.  I'll leave that exercise for others.

**Reply by GrampaWildWilly** [reply]
What is it with you guys?  Post your system environment.  I did as the first thing in my post.  You need to do at least that, or follow the instructions in the link Paul posted.  It's important information.  You need to post it.  Always.  We wouldn't always ask for it if it weren't important.  Post it.

**Comment by git7878** [comment]
Paul,
I should mention that this problem started only after the last
video-downloadhelper update received a few days ago.  Before that
everything was fine.
Tom

**Comment by bumsbums** [comment]
Hi Paul,
I have the same problem with the same website. Here's the info as shown in your link:

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: de
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Ping Pong Paul",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}



The URL of the video:

https://www.eroprofile.com/m/videos/view/tiktokkaajaliisa


Thanks in advance!

**Comment by git7878** [comment]
Paul,
I should also add that the colored balls do not light up and that "no media
to process in the current tab..." is the only message seen after clicking
on them - there are
no choices at all.
Sorry about the piecemeal messages but I've been battling Covid for the
past few days.
Thanks again,
Tom.

**Reply by GrampaWildWilly** [reply]
May the COVID not be with you . . . and soon.  Perhaps it has led you to miss the fact that you are not running the same releases of the software that I am.  Or maybe you missed my post above altogether.

You are having problems.  I am not . . .  Well, I am having problems that are easily circumvented.  You want to click the Wiki link at the top of this page & look for references to where you can get the CoApp & where you can get the beta versions of the browser extension.  Once you are using the same versions I am, dollars to donuts you'll get the same mostly good results I am.

**Comment by git7878** [comment]
It works fine now!  Thank you for all your hard work and generosity.
Tom

**Comment by bumsbums** [comment]
Thank you so much for your help! It's working again!

**Comment by bumsbums** [comment]
BTW, couold you also adjust it so the downloads show up in the Firefox downloads section? That would be fantastic! Thank you in advance for all your great work!

**Reply by GrampaWildWilly** [reply]
The only way that information is going to appear in the Firefox download manager is if you tell VDH to not use the CoApp.  Then VDH will use Firefox itself to do the download, which has many drawbacks.  On the other hand, in the SIX MONTHS since you posted this comment, VDH has changed a lot.  VDH now keeps a history log of your downloads.  That probably serves the purpose you were looking for by asking for the downloads to appear in the Firefox download manager.

**Comment by Carciofo70** [comment]
@paulrouget

the problem is related to the site https://thisvid.com
in the same way it happened at the beginning of the year with eroprofile.com, as described in my first post. I tried with different browsers and different versions of VDH even on different PCs but the result remains the one described in the figure in the red box, whatever video I intend to download. I hope for a simple solution like the previous time. Thank you!

**Reply by Carciofo70** [reply]
@paulrouget 


![xo27hgtt](https://github.com/user-attachments/assets/8bf4d954-fd27-4a4a-ad6d-1587b24a7f62)

**Reply by Mattmjs** [reply]
I see it as http media 88Mb like @GrampaWildWilly. On stable extension, Windows 10 in Firefox & Ungoogled chromium.

**Reply by Carciofo70** [reply]
so how come on 3 different "clean" PCs with Firefox, Win10 and stable VDH it always behaves the same way with me?

**Reply by GrampaWildWilly** [reply]
I can't answer that.  My computer has done things I can't explain.  I've even found solutions to some problems that make no sense.  So some things are just beyond the understanding of mere mortal users.

I do note that one of the tests I ran on Ms Wet Pants was showing things that looked like it was a livestream.  In my test, that variant did not show a duration on the VDH menu at all.  You have managed to get it to show a duration of 6 seconds.  That is obviously incorrect.  But if you consider it as a livestream, that could be correct.  The 6 seconds is only the beginning of the not-live livestream.  After VDH downloads those 6 seconds, I would hope that VDH continues & downloads the next bit of the video, perhaps another 6 seconds.  VDH will then stop displaying a percentage of the file that has been downloaded so far.  Instead, it will start showing an elapsed time timer indicating how long it's been downloading.  With a livestream, you don't know the duration of the show.  I know, this isn't a livestream.  But somehow VDH is recognizing it as a livestream, for whatever reasons.  Here is a blowup of a section of one of the images I posted from my test on August 26:

![#01](https://github.com/user-attachments/assets/c87d33a2-b8c3-4bf9-96f1-2535bc1da169)

The little indicator I'm pointing out in that image is VDH's way of saying it is recognizing the item as a livestream.  Note also that it doesn't say what percentage of the video it has downloaded so far.  It says simply that it had been downloading the video for 34 seconds at the time I took that image.  The download did proceed to completion after I took that screenshot, despite the fact that VDH somehow did not recognize a duration.  In your case, even though it says it's only 6 seconds long, what happens when you try to download the video anyway?  Does it download the 6 seconds then continue on?

**Reply by Carciofo70** [reply]
No, only 6 seconds...

**Reply by GrampaWildWilly** [reply]
You haven't posted your `Copy information to clipboard` from the VDH settings.  Please do that.

**Reply by Carciofo70** [reply]
version: 9.1.0.42
channel: stable
build date: 2024-08-02
build options: linuxlic: false, noyt: false, browser: firefox
lang: it
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\3350",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Reply by Carciofo70** [reply]
version: 9.1.0.42
channel: stable
build date: 2024-08-02
build options: linuxlic: false, noyt: false, browser: firefox
lang: it
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\3350",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Carciofo70** [comment]
![immagine](https://github.com/user-attachments/assets/48633c4d-f5a8-41e3-a089-b94ebc4d4a9e)

**Reply by GrampaWildWilly** [reply]
![#01](https://github.com/user-attachments/assets/0e104b86-222e-4bd8-bfa2-0fd5953a355a)
![#02](https://github.com/user-attachments/assets/a20898ab-465c-4a7e-af83-513a5cb143b4)
![#03](https://github.com/user-attachments/assets/a4ed2296-f5d3-4cc4-9450-e2c5cfdcb3a3)
![#04](https://github.com/user-attachments/assets/ac5ebee7-d99e-4808-8523-59c4d3046033)
![#05](https://github.com/user-attachments/assets/51d1eb8e-f761-4408-bd7f-03067a7592a3)
![#06](https://github.com/user-attachments/assets/54cf57aa-8c37-44c2-94c7-eb12bf8a80d8)
![#07](https://github.com/user-attachments/assets/b72952d5-2d75-40d5-aa6e-7b1d40d8031c)

I don't think you are accomplishing anything by posting more & more examples that you are having trouble with.  We get the idea that you are having trouble downloading from this site.  I think we are also getting the idea that I am having no trouble downloading from this site.  Let's leave it be for now & just wait for @paulrouget Paul to address the issue.

**Reply by GrampaWildWilly** [reply]
I also note that, despite my repeated exhortations for you to install the VDH beta, you have not done so.  Let me repeat.

INSTALL THE BETA.
